packages/core/src/components/hotkeys/hotkeyParser.ts

```diff
@@ -61,18 +61,9 @@ export const CONFIG_ALIASES: KeyMap = {
     space: " ",
 };
 
+/** Key mapping used in getKeyCombo() implementation for physical keys which are not alphabet characters or digits. */
 export const SHIFT_KEYS: KeyMap = {
     "~": "`",
-    "!": "1",
```

Comment:
We no longer need the 0-9 digit keys in this mapping because we now handle events with `{ code: "DigitX" }` in a more straightforward way. In a future PR, we can remove this `KeyMap` entirely by switching to `event.code` for more keys, but for now I decided to be conservative and make fewer behavioral changes in this PR.